BasicSample/Assets/AppRemoting/Scripts/AppRemotingSample.cs

```diff
@@ -133,15 +165,41 @@ public void ConnectToRemote(string address = null)
             StartCoroutine(Remoting.AppRemoting.Connect(remotingConfiguration));
         }
 
+        /// <summary>
+        /// Listens to the incoming connections as specified in the remotinglistenconfiguration
+        /// </summary>
+        public void ListenToRemote()
+        {
+            m_appRemotingMode = AppRemotingMode.listen;
+            StartCoroutine(Remoting.AppRemoting.Listen(remotingListenConfiguration));
+        }
+
         /// <summary>
         /// Disconnects from the remote session.
         /// </summary>
         public void DisconnectFromRemote()
```

Comment:
Can we add a "disconnect" button to the flat UI?  I found i cannot easily stop the "listen mode" if the connection never happens.